tests/utils.go

```diff
@@ -42,6 +55,16 @@ func CDIFailHandler(message string, callerSkip ...int) {
 	Fail(message, callerSkip...)
 }
 
+func GetCDIClient() *clientset.Clientset {
```

Comment:
How is this result any different from GetKubeClient below?

Reply:
the cdi client is different from the kubernetes client.

cdi client knows about cdi objects (like DataVolumes) and the kubernetes client only knows about k8s core objects (like pods, pvcs, etc...) They are separate objects.

Reply:
Okay maybe I am missing something, but both use the same master and kubeconfig parameters. And they call the same functions to create the ClientSet, so they should be identical when we are running the test. Well except that the GetKubeClient adds some api stuff.

Reply:
>  And they call the same functions to create the ClientSet

cdiClient is calling a package that's in the cdi src tree. Take a closer look at the clientset import at the top of the file.

Reply:
I understand, I totally missed that little difference. Thanks, the Framework patch I will push shortly will have both kubeclient and cdiclient available by default.